test/scenarios-bootc/presubmits/el96-src@ai-model-serving-online.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# Currently, RHOAI is only available for x86_64
+check_platform() {
+    local -r record_junit=${1:-false}
+
+    if [[ "${UNAME_M}" =~ aarch64 ]]; then
+        if "${record_junit}"; then
+            record_junit "setup" "scenario_create_vms" "SKIPPED"
+        fi
+        exit 0
+    fi
+}
+
+scenario_create_vms() {
+    check_platform true
+
+    # Increased disk size because of the additional embedded images (especially OVMS which is ~3.5GiB)
```

Comment:
I think we need to make that comment happen:
`write /var/lib/microshift/etcd/member/snap/db: no space left on device`

```
Mar 19 12:41:18 ip-192-168-122-80.us-west-2.compute.internal microshift[7967]: {"level":"fatal","ts":"2025-03-19T12:41:18.199239Z","caller":"backend/batch_tx.go:282","msg":"failed to commit tx","error":"write /var/lib/microshift/etcd/member/snap/db: no space left on device","stacktrace":"..."}
```